lib/ohai/plugins/linux/ipc.rb

```diff
@@ -0,0 +1,49 @@
+#
+# Author:: Jay Vana <jsvana@fb.com>
+# Copyright:: Copyright (c) 2016 Facebook, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:IPC) do
```

Comment:
We'll want this to be an optional plugin as well

```suggestion
Ohai.plugin(:IPC) do
  optional true
```